roles/splunk_common/tasks/set_kvstore_port.yml

```diff
@@ -8,3 +8,9 @@
     owner: "{{ splunk.user }}"
     group: "{{ splunk.group }}"
 
+- name: Adjust server.conf permissions (Windows)
```

Comment:
This might need to be a separate play because ideally you'll always want this to touch server.conf before Splunk starts, but there's no guarantee `set_kvstore_port.yml` will continue to be the last play included before the start operation.